packages/lit-renderer/src/lit-renderer.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { ElementPart, nothing, RenderOptions, TemplateResult } from 'lit';
+import { AsyncDirective } from 'lit/async-directive.js';
+
+export type LitRenderer = (...args: any[]) => TemplateResult;
```

Comment:
```suggestion
export type LitRenderer = (...args: unknown[]) => TemplateResult;
```

Reply:
Tried this in the prototype project and got TS compilation errors e.g.

```
src/combo-box-renderer.ts:14:3 - error TS2344: Type 'ComboBoxLitRenderer<unknown>' does not satisfy the constraint 'AbstractLitRenderer'.
  Types of parameters 'model' and 'args' are incompatible.
    Type 'unknown' is not assignable to type 'ComboBoxItemModel<unknown>'.

14   ComboBoxLitRenderer<unknown>
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```